src/NHSD.BuyingCatalogue.Identity.Api/Controllers/AccountController.cs

```diff
@@ -55,7 +55,7 @@ public async Task<IActionResult> Login(LoginViewModel viewModel)
 
             if (!signInResult.IsSuccessful)
             {
-                ModelState.AddModelError(string.Empty, SignInErrorMessage);
+                ModelState.AddModelError("LoginError", SignInErrorMessage);
```

Comment:
for sure.

When we get a failed sign in result, we need to display the error associated with that, and then as part of the error summary that error needs to link to the Email field. This is a particularly awkward bit of behaviour when you're just assigning the error as a model error with no associated property.

To avoid that, I've created a new property to assign the error to, and a custom attribute to tell that property to link to Email when clicked, instead of to itself. This means that here, instead of applying the error message to the model as a whole, we assign it to our new property, which gets it displayed correctly, and then the attribute changes its link location to be EmailAddress.